lib/bitcoin/BitcoinClient.ts

```diff
@@ -13,33 +17,91 @@ export default class BitcoinClient implements IBitcoinClient {
   /** Bitcoin peer's RPC basic authorization credentials */
   private readonly bitcoinAuthorization?: string;
 
+  /** Wallet private key */
+  private readonly privateKey: PrivateKey;
+  private readonly privateKeyAddress: Address;

```

Comment:
Consider renaming to just `address` or `walletAddress`, since address is really derived from the "public" portion of the key.

Reply:
The address is actually the private key's address so I'll keep the name.